en/starter/settingup.md

```diff
@@ -19,7 +19,7 @@ redirect_from: "/starter/setting up.html"
 
 ## macOS
 
-<span class="arrow">&#8227;</span> Download and install [Xcode 8.2](https://developer.apple.com/download/).
+<span class="arrow">&#8227;</span> Download and install [Xcode 8.3](https://developer.apple.com/download/).
```

Comment:
I just tested and Kitura 1.6 works with Xcode 8.3 and Swift 3.1. However, the test that tests for server failure fails, caused by a BlueSocket update. This was fixed by changing the test in https://github.com/IBM-Swift/Kitura/issues/1052.